About and Donate

mcconnellm opened the pull request:
About and Donate views models, and templates added.

mcconnellm committed:
About and Donate

About and Donate views models, and templates added.

jjeffryes commented on js/router.js:
```diff
@@ -26,14 +28,14 @@ module.exports = Backbone.Router.extend({
     "about": "about",
-    "support": "support"
+    "donate": "donate",
+    "support": "donate"
```
Support and donate are the same thing, you can just have one or the other.

mcconnellm committed:
Merge remote-tracking branch 'refs/remotes/OpenBazaar/master'

mcconnellm committed:
Merge remote-tracking branch 'refs/remotes/OpenBazaar/master'

mcconnellm committed:
Revised About/Donate

Various bug fixes.

mcconnellm committed:
Revised templates.

Templates now have barebones content.

jjeffryes committed:
Patches

- fixed rendering of aboutVw
- fixed rendering of donateVw
- fixed router for donate
- added some padding to about and donate templates

mcconnellm committed:
Merge pull request #1 from mcconnellm/joshRevisions2

Patches to About and Donate

jjeffryes commented:
For some reason this is out of sync with the master repo even after a rebase. Closing this PR while I fix it.
